learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+
+## Requirements
+
+You must have a Meilisearch Cloud account to have access to search analytics.
+
+## Activating analytics
+
+### Enable analytics in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics" section and click on the "Enable analytics" button:
+
+![The analytics section of the project overview. It shows one button, "Enable analytics", and a short explanation of the feature.](https://raw.githubusercontent.com/meilisearch/documentation/cloud-analytics-guide/assets/images/cloud-analytics/01-analytics-enable.png)
+
+Meilisearch Cloud will begin processing your request. The "Analytics" section will update when the feature is enabled.
+
+### Update URLs in your application
+
+Enabling analytics generates a new URL for your project. Meilisearch Cloud is only able to track metrics for queries sent to this URL. 
+
+Update your application so all API requests point to the new URL:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com/indexes/products/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "green socks" }'
+```
+
+Your old project URL will remain functional, but API calls using the old URL will not be reflected in the analytics interface.
+
+<Capsule intent="note" title="Analytics and latency">
+To capture and process search analytics data, Meilisearch Cloud must route API requests through a proxy. This might increase latency in some applications.
+</Capsule>
+
+### Configuring click-through rate and average click position
+
+To track metrics like clickthrough rate and average click position, Meilisearch Cloud needs to know when users click on search results.
+
+Every time a user clicks on a search result, your application must send a `click` event to the `POST` endpoint of Meilisearch Cloud analytics route:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer Meilisearch_API_Key'
+  --data-binary '{ 
+    "eventType": "click",
+    "eventName": "search result click",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+#### The `click` event object
+
+The `click` event must deliver an object with the following fields:
+
+```json
+{
+  "eventType": "click",
+  "eventName": "search result click",
+  "indexUid": "products",
+  "objectID": "0",
+  "position": 0
+}
+```
+
+- `eventType`: a string indicating this is a `click` event
+- `eventName`: a string describing the event
+- `indexUid`: a string indicating the clicked document's index
+- `objectID`: a string indicating the clicked document's primary key
+- `position`: an integer indicating the clicked document's position in the list of search results
+
+### Configuring conversion rate
+
+To track conversion rate, first identify what should count as a conversion for your application. For example, in a web shop, a conversion might be a user finalizing the checkout process. 
```

Comment:
there can be multiple conversion events. using the webshop example we can give two examples: 'add to card' and 'checkout'

Reply:
Do we support multiple conversion events, though? How does Meilisearch parse multiple conversion events? @brunoocasali